templates/http-py/content/app.py

```diff
@@ -1,5 +1,6 @@
 from spin_http import Response
 
+
```

Comment:
What's the reason for the extra empty line?

Reply:
It is to conform with the pep8 style guide.